src/models/configs.ts

```diff
@@ -33,4 +41,5 @@ export interface ReportPortalConfig {
   isLaunchMergeRequired?: boolean;
   skippedIssue?: boolean;
   includeTestSteps?: boolean;
+  restClientConfig?: RestClientConfig;
```

Comment:
Can we use `AxiosRequestConfig` from `axios` as the `restClientConfig` supports overriding others rest client defaults?
Later it will be incapsulated into `RestClientConfig` interface from `client-javascript` when it will be migrated to TS.

Reply:
Hi, correct me please if i am wrong, to use AxiosRequestConfig i have to add axios as dependency to agent-js-playwright?